### PR TITLE
Quiz 08 on spread

### PR DESCRIPTION
**Pull request by alayek**
Corresponding to `08_spread.test.js` in templates. Refer #20

### CODE REVIEW

**Review by kentcdodds — CHANGES_REQUESTED** [review]
With this one could we get existing ES5 implementations that we could refactor? I can do it myself if you prefer.

**Comment by alayek** [comment]
@kentcdodds Ok, I will fix this one. 👍

File: quizzes/08_spread.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+console.log(
+  // getMax([7, 8, 5, 9, 20, 7])
```
Comment (kentcdodds) [inline quizzes/08_spread.js]: I'd prefer to not pass arguments to the quiz functions and just have them declare the variables inside the function itself.

Comment (alayek) [reply quizzes/08_spread.js]: ok, alright.

Comment (kentcdodds) [reply quizzes/08_spread.js]: Thank you very much for your willingness to iterate with me on this stuff. I hope you're having a good time and learning stuff!

Comment (alayek) [reply quizzes/08_spread.js]: Yes absolutely! I am also on the gitter chat room of this repo. Kindly feel free to tag me there if you need to communicate anything that is better suited in chat 😅

**Review by kentcdodds — APPROVED** [review]
Excellent!

**Pull request merged**